examples/mesh-update.rs

```diff
@@ -0,0 +1,167 @@
+extern crate cgmath;
```

Comment:
This example is too complicated. I would like to see either smaller one, or none at all.

Reply:
Yes, I agree. I'll re-implement with a tetrahedron instead of a cube.